src/lib/core/utils/number-decimal-point.ts

```diff
@@ -0,0 +1,9 @@
+// util to return a number with specific decimal points
+export function numberDecimalPoint(
+  value: number,
+  decimalPoint: number
+): number {
+  return Number(
+    Math.round(parseFloat(value + 'e' + decimalPoint)) + 'e-' + decimalPoint
```

Comment:
Hi @moontrip 

I'm worried that this function will fail for numbers that get rendered to "scientific/exponential notation" as strings.

For example, 
```
numberDecimalPoint(500000000000000000, 3)
500000000000000000
numberDecimalPoint(5000000000000000000, 3)
NaN
```
Would something like `Number(value.toPrecision(decimalPoint))` be more robust?

Reply:
Except maybe `toFixed` !!